cmake/Modules/FindSYCL/run_sycl.cmake

```diff
@@ -108,10 +113,16 @@ SYCL_execute_process(
   )
 
 # Generate the code
+set(SYCL_DEPENDENCY_FLAGS "")
+# TODO: enable incremental build on Windows
```

Comment:
If there is no dependency checking, it is dangerous for compilation, when dependency changes. So maybe you can do something after the compilation to give a hint, all files are always re-compiled, when rebuilding.

Reply:
Thanks, enabled incremental build on Windows.